<launch>
  <!-- camera driver -->
  <node name="dvs_ros_driver" pkg="dvs_ros_driver" type="dvs_ros_driver">
	<rosparam command="load" file="$(find dvs_ros_driver)/config/fast.yaml" />
  </node>

  <!-- visualization -->
  <node name="dvs_renderer" pkg="dvs_renderer" type="dvs_renderer">
    <!-- <param name="display_method" value="grayscale"/> -->
    <param name="display_method" value="red-blue"/>
    <remap from="events" to="/dvs/events" />
    <remap from="camera_info" to="/dvs/camera_info" />
  </node>

  <!-- display -->
  	  <node name="image_view" pkg="image_view" type="image_view"> 
    	 <remap from="image" to="dvs_rendering"/> 
  	  </node> 

  	<!--  <node name="image_view2" pkg="image_view" type="image_view"> -->
    <!--	<remap from="image" to="dvs_undistorted"/> -->
  	<!--  </node> -->
	 
  <!-- configure -->
  <!-- <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" /> -->

</launch>
